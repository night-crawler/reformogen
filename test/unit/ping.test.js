

it('should render ping-pong', () => {
    const wrapper = shallow(
        <label>Ping? Pong!</label>
    );
    expect(wrapper).toMatchSnapshot();
});
