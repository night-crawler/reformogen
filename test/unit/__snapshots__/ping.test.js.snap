// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ping-pong 1`] = `
<label>
  Ping? Pong!
</label>
`;
